src/net.h

```diff
@@ -972,7 +988,7 @@ class CConnman
     /**
      * Return vector of current BLOCK_RELAY peers.
```

Comment:
Doing this rename would touch 20 lines of code in 8 files, but could be done with a scripted diff. I agree it would be more consistent, so personally, I'm in favor, but I'd also understand not wanting to increase the diff that much.

Reply:
I agree that it would be more consistent and added a commit to do this - I'll check how many additional conflicts this introduces...